packages/core/handlers/routers/user.js

```diff
@@ -13,8 +32,8 @@ router.route('/user/login').post(
             'username',
             'password',
         ]);
-        const user = await User.loginUser({ username, password });
-        const token = await user.createUserToken(120);
+        const user = await loginUser.execute({ username, password });
```

Comment:
Apologies this is just me making sure I understand the hexagonal basics, but... .execute is basically the only method we'd call on every use case class, yeah? When do other methods get called?